Equipment/SetBonusControl.cpp

```diff
@@ -242,6 +243,28 @@ void SetBonusControl::equip_item(const int item_id) {
             break;
         }
     }
+    else if (set_name == "Stormshroud Armor") {
+        switch (num_pieces) {
+        case 2:
+            if (!active_procs.contains("STORMSHROUD_2P_GAIN"))
+                active_procs["STORMSHROUD_2P_GAIN"] = new InstantSpellProc(pchar,
+                                                                        "Stormshroud 2 set",
+                                                                        "Assets/items/Inv_shoulder_05.png",
+                                                                        {ProcInfo::MainhandSpell, ProcInfo::MainhandSwing, ProcInfo::OffhandSwing},
+                                                                        0.05, MagicSchool::Nature, 15, 25, 0.0, ConsumeCharge::No);
```

Comment:
Do we know for certain Stormshroud 2p proc doesn't consume e.g. Stormstrike charges?

Reply:
Honestly, I do not know. It would be nice if someone could test it in the game.